### PR TITLE
Changed bad-by-flat threshold to match MATLAB PREP

### LINKED ISSUE #59: "Bad by flat" threshold differs between PyPREP and Matlab PREP
In Matlab PREP, the criteria for a "flat" channel are that it either has a median absolute deviation below `10e-10` or a standard deviation below `10e-10`.

In PyPREP this is mis-copied over as `1e-10`, but there's an additional difference in that MNE and EEGLAB scale their raw data values differently, such that the first 2 samples in MATLAB are

`[-3.7068, -4.8012, ...]`

whereas in MNE they're

`[-3.70680070e-06, -4.80121040e-06, ...]`. 

Unless I'm doing the math wrong in my head, the correct equivalent threshold to use for PyPREP would therefore be `10e-16`.

### PR DESCRIPTION
**Pull request by a-hurst**
# PR Description

Closes #59.

Basically, this PR just corrects for the fact that EEGLAB's internal units are in micro volts whereas MNE's internal units are in volts.

# Merge Checklist

- [ ] the PR has been reviewed and all comments are resolved
- [ ] all [CI][what-is-ci] checks pass
- [ ] (if applicable): the PR description includes the phrase `closes #<issue-number>` to [automatically close an issue][auto-close-documentation]
- [ ] (if applicable): bug fixes, new features, or [API][what-is-api] changes are documented in [whats_new.rst][whats-new-file]

[what-is-ci]: https://help.github.com/en/actions/building-and-testing-code-with-continuous-integration/about-continuous-integration
[auto-close-documentation]: https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword
[what-is-api]: https://en.wikipedia.org/wiki/Application_programming_interface
[whats-new-file]: https://github.com/sappelhoff/pyprep/blob/master/docs/whats_new.rst

### CODE REVIEW

File: pyprep/find_noisy_channels.py [hunk 1]
```diff
@@ -154,12 +154,13 @@ def find_bad_by_nan_flat(self):
         nan_channel_mask = [False] * self.original_dimensions[0]
         no_signal_channel_mask = [False] * self.original_dimensions[0]
 
+        flat_threshold = 1e-15  # corresponds to 10e-10 µV in MATLAB PREP
```
Comment (sappelhoff) [inline pyprep/find_noisy_channels.py:157]:
```suggestion
        FLAT_THRESHOLD = 1e-15  # corresponds to 10e-10 µV in MATLAB PREP
```

File: pyprep/find_noisy_channels.py [hunk 2]
```diff
@@ -154,12 +154,13 @@ def find_bad_by_nan_flat(self):
...
         for i in range(0, self.original_dimensions[0]):
             nan_channel_mask[i] = np.sum(np.isnan(self.EEGData[i, :])) > 0
         for i in range(0, self.original_dimensions[0]):
             no_signal_channel_mask[i] = (
-                robust.mad(self.EEGData[i, :], c=1) < 1e-10
-                or np.std(self.EEGData[i, :]) < 1e-10
+                robust.mad(self.EEGData[i, :], c=1) < flat_threshold
+                or np.std(self.EEGData[i, :]) < flat_threshold
```
Comment (sappelhoff) [inline pyprep/find_noisy_channels.py:163]:
```suggestion
                robust.mad(self.EEGData[i, :], c=1) < FLAT_THRESHOLD
                or np.std(self.EEGData[i, :]) < FLAT_THRESHOLD
```

**Review by sappelhoff — APPROVED** [review]
Thanks :-) 

just a style suggestion: When a constant like "flat_threshold" is introduced, I like to make this upper case only to clearly indicate that this is a constant.

**Comment by a-hurst** [comment]
> just a style suggestion: When a constant like "flat_threshold" is introduced, I like to make this upper case only to clearly indicate that this is a constant.

Fixed in the latest commit!

**Comment by sappelhoff** [comment]
Thanks!

**Pull request merged**